deploy/config.py

```diff
@@ -0,0 +1,11 @@
+# This file needs to be copied to /app/ga4gh/config.py by default
+import os
+
+#TODO this logic could move to frontend.configure() or BaseConfig
+# For docker, if/when a default is set in serverconfig.py, run -v /localdata:/default-path
+# If the env variable GA4GH_DATA_SOURCE is set, use that path. Otherwise, use the default path
+DATA_SOURCE = os.getenv('GA4GH_DATA_SOURCE', "/ga4gh-example-data")
```

Comment:
The only environment variable imported is GA4GH_CONFIGURATION, the path of the config file. I agree we should support this in frontend, but I thought that would be out of scope of adding support for Docker. This allows setting the location of the datasource by passing an env variable to the docker container. os.getenv returns arg2 if arg1 is unset. This logic could be discarded when frontend.py is updated with production oriented argument parsing. Res: won't fix